.rubocop.yml

```
@@ -0,0 +1,8 @@
+require:
+  - rubocop-capybara
+  - rubocop-rails
+
+AllCops:
+  SuggestExtensions: false
+Metrics/BlockLength:
+  Max: 100
```

Can you replace with
```suggestion
inherit_from:
  - https://raw.githubusercontent.com/wtag/default-configs/master/rubocop-default-config-ruby2.6.yml

```

---

Done apu